core/src/main/java/Scenes/Leaderboards.java

```diff
@@ -60,15 +52,25 @@ public void render(float delta) {
     spriteBatch.end();
   }
 
-  @Override
-  public boolean tap(float x, float y, int count, int button) {
-    float worldX = x * WidthWorldPixelRatio;
-    float correctedY = Gdx.graphics.getHeight() - y;
-    float worldY = correctedY * HeightWorldPixelRatio;
-    if (backButton.isClicked(worldX, worldY)) {
-      System.out.println("Go back to main menu");
-      mainGame.setScreen(FilePlayMain.ScreenType.MAINMENU);
-    }
-    return false;
+  /**
+   * This adds listeners to each of the buttons along with what to do upon being touched. A new
+   * InputListener is added inline overriding the touchDown method to determine what to do when
+   * touched.
+   */
+  private void addAllListeners() {
```

Comment:
Hey James, I fixed the viewing on the emulator. Everything should be fitting nicely now. The desktop version should not be resized. If you resize it whether by stretching the sides or hitting the full screen button, LibGDX does not process this as a resizing therefore the resize method does not get called. What this does is make the button locations wrong until you actually hit the location of a button in which a screen is changed and _then_ the resize method is called. You can confirm this by having a print statement in BaseScreen's resize method and see that resize does not get called until a screen transition is made. We can fix this by calling resize in the render method however this does not seem efficient and is why I left it out. Confirm that the game runs well on the emulator on your end once more before merging please.